package.json

```diff
@@ -33,8 +33,8 @@
     "@jest/types": "^29.6.3",
     "@swc/jest": "^0.2.36",
     "@types/jest": "^29.5.12",
-    "@typescript-eslint/eslint-plugin": "^7.3.1",
-    "@typescript-eslint/parser": "^7.3.1",
+    "@typescript-eslint/eslint-plugin": "7.2",
+    "@typescript-eslint/parser": "7.2",
```

Comment:
Need to revisit before merging - I pinned this because a node version issue which was described here: https://interledgerfoundation.slack.com/archives/C06KZCKEEKW/p1710798244722689

Reply:
I figured this would go away when the node 20 upgrade happens (soon?). Otherwise it seems like we can alternatively upgrade to a new node 18 version (not sure what all we would need to specify for that).

Reply:
waiting on that one is probably a good idea